### ISSUE TITLE
Debug build failed

### ISSUE THREAD

**Issue by noavarice** [texstudio-org/texstudio#16]
### Description
For some reason, 'Debug' build constantly fails because of `git_revision.sh` script (while 'Release' build works just fine). Exact error looks as follows:
> git_revision.sh:12: cannot create src/git_revision.cpp: Directory nonexistent

Running `git_revision.sh` outside of build process (`./git_revision.sh g++ <output_dir>`) tells that there are not enough permissions for creating specified file despite of the fact that all possible permissions are given. Even manually setting permissions (through `chmod 777 git_revision.sh`) does not do the trick
### Software information
OS: Kubuntu 17.10
Qt version: 5.9.1

**Comment by sunderme** [comment]
I assume that you don't build in the "texstudio" directory ?

**Comment by noavarice** [comment]
Yes, build goes inside directory which at the same level with texstudio dir

**Comment by sunderme** [comment]
please try out the latest commits

**Comment by noavarice** [comment]
So, I cloned repository again and it successfully compiled from `texstudio` directory:
```
qmake
make -j4
```
After making this, I created `debug` directory at the same level with `texstudio` and specified this directory to be a build dir for Qt, and all this works fine.
The only thing that a bit annoying me - `git_revision.o` file that appears after each build process. I suppose that you have forgotten to `.gitiignore` it. Point if I am wrong
Also thaks for help in this silly problem)
